__mocks__/@ui5/AnalyticalTable.js

```
@@ -0,0 +1,63 @@
+import _ from 'lodash';
+
+const React = require('react');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)